tests/test_jinja.py

```diff
@@ -61,6 +61,140 @@ def test_jinja_platform(monkeypatch):
     assert render_jinja(template, filename="", is_included=False) == platform.python_revision()
 
 
+def test_jinja_x86(monkeypatch):
+    template = "{{ x86 }}"
+
+    monkeypatch.setattr(platform, 'machine', lambda: 'x86')
+    assert render_jinja(template, filename="", is_included=False) == 'True'
+
+    monkeypatch.setattr(platform, 'machine', lambda: 'x86_64')
+    assert render_jinja(template, filename="", is_included=False) == 'False'
+
+
+def test_jinja_x86_64(monkeypatch):
+    template = "{{ x86_64 }}"
+
+    monkeypatch.setattr(platform, 'machine', lambda: 'x86')
+    assert render_jinja(template, filename="", is_included=False) == 'False'
+
+    monkeypatch.setattr(platform, 'machine', lambda: 'x86_64')
+    assert render_jinja(template, filename="", is_included=False) == 'True'
+
+
+def test_jinja_linux(monkeypatch):
+    template = "{{ linux }}"
+
+    monkeypatch.setattr(sys, 'platform', 'linux')
+    assert render_jinja(template, filename="", is_included=False) == 'True'
+
+    monkeypatch.setattr(sys, 'platform', 'win')
+    assert render_jinja(template, filename="", is_included=False) == 'False'
+
+    monkeypatch.setattr(sys, 'platform', 'darwin')
+    assert render_jinja(template, filename="", is_included=False) == 'False'
+
+
+def test_jinja_linux32(monkeypatch):
+    template = "{{ linux32 }}"
+
+    monkeypatch.setattr(sys, 'platform', 'linux')
+
+    monkeypatch.setattr(platform, 'architecture', lambda: '32bit')
```

Comment:
`platform.architecture` returns a tuple:

```python
is32bit = '32bit' == platform.architecture()[0]
```

So this line should be:

```python
monkeypatch.setattr(platform, 'architecture', lambda: ('32bit',))
```

👍

Reply:
Almost there. How do I perform the tests for conda-devenv in my machine, so I don't need to keep on this trial and error with CI?

Reply:
Please see [this section](https://github.com/ESSS/conda-devenv/blob/master/CONTRIBUTING.rst#get-started) in the contributing docs. It is a little hidden I suppose...